CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.7.1 (Unreleased)
```

Comment:
Hvis man bruker `?.?.?` som versjon, så vil `bumpversion` fylle inn riktig versjonsnummer når releasen lages. Nå ligger det f.eks. nye ting på `master` som tilsier at neste release kanskje blir en minor release i stedet for en patch release.